shop/src/main/java/ro/msg/learning/shop/domain/EntityWithUUID.java

```diff
@@ -0,0 +1,19 @@
+package ro.msg.learning.shop.domain;
+
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Data;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.UUID;
+@Data
```

Comment:
One empty line between import and annotations + only one line between package definition and imports (not 2). 

Most of the issues will be automatically resolved when you reformat the code using the IntelliJ built-in function.